migration/servicedmigration/service.py

```diff
@@ -0,0 +1,39 @@
+
+import sys
+import json
+
+from version import versioned
+
+
+class Service:
+    """
+    Manages loading, altering, and commiting services.
+    """
+
+    @versioned
+    def __init__(self):
```

Comment:
I think __init__ should take a string as input parameter, rather than reading the filename directly from sys.argv.  Better yet, the string should be the json formatted string.  Perhaps even better would be __init__ take no parameters, but have a separate layer that deserializes a json string and construct the Service object.

I anticipate that the json file will eventually [contain the entire service tree](https://github.com/control-center/serviced/pull/1613#issuecomment-76758816), not just one service.  The proposed refactoring would ease that transition.